queries/reflect.go

```diff
@@ -158,10 +174,95 @@ func (q *Query) Bind(ctx context.Context, exec boil.Executor, obj interface{}) e
 		return errors.Wrap(err, "error from rows in bind")
 	}
 
-	if len(q.load) != 0 {
+	if len(q.load) != 0 && !q.loadJoin {
 		return eagerLoad(ctx, exec, q.load, q.loadMods, obj, bkind)
 	}
 
+	if q.loadJoin {
```

Comment:
I wish this could be a bit more generic and wasn't a big special case everywhere. If we were to do this properly I don't think we'd have to make a large distinction, it would only be a difference in how we look up the data.

Reply:
Yes, I understand. If we did a wholesale change to using joined eager joins everywhere, trying to do both together probably wouldn't be needed. I personally think there is some risk in trying to change it all at once, and that having both in parallel side by side may be useful - to people who depend on the current behavior, or just to bake the new approach over time - but it's all good. Thanks again.